usagov/vote-gov-drupal
# Vote-943 -> Site Wide Banner test

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[vote-943](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-943&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This Pr will add test coverage for the sitewide banner... this will check the access for each role and their ability to add the banner along with deleting it as well.

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:open`
5. select `sitewide-banner-access.cy.js` test and verify that test is passing
